doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -368,6 +368,7 @@ Cellular samples
 * :ref:`modem_shell_application` sample:
 
   * Updated to use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
+  * Added the ``uiccpowersave`` link command for reading/setting UICC power saving mode. This feature is supported by Modem Firmware 2.0.2 or higher.
```

Comment:
```suggestion
  * Added the ``uiccpowersave`` link command for reading or setting UICC power saving mode. 
    This feature is supported by modem firmware v2.0.2 or higher.
  * Updated to use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
```
line no: 370 and 371 could be interchanged to follow the order Added, updated.